pkg/permissions/permissions.go

```diff
@@ -0,0 +1,40 @@
+package permissions
+
```

Comment:
Whats the reason for moving this into a new pkg?

Reply:
I added `AppPermissionsManager` into this package so it wasn't just sources. don't feel strongly can move it back